### DISCUSSION TITLE
How to make sure the project is "web scrapable"?

### DISCUSSION THREAD

**Discussion by faaizalikhan1** [vercel/next.js#12875] [Help]
Hi, I have been trying to learn nextjs for a project, I have been told to make sure the project is web scrapable, and has dynamic titles, from h1- h4, I read up on next/head and was able to integrate it for SEO purposes, but I was told its still not fully web scrapable. I don't know much about this and the google searches are confusing me, I'd appreciate if anyone could help me in this, what else should I include in the project?

**Comment by leerob** [comment]
My assumption is that "web scrapable" means good [on-page SEO](https://moz.com/blog/category/on-page-seo). When a search engine crawler "scrapes" your page, you want to ensure you have the proper `meta` tags and heading structure.

Next.js shouldn't matter for this. If you want to simplify adding `meta` tags, check out [next-seo](https://github.com/garmeeh/next-seo).
